src/sea_ice_model.jl

```diff
@@ -0,0 +1,117 @@
+using Oceananigans.Fields: TracerFields
+using ClimaSeaIce.SeaIceThermodynamics: external_top_heat_flux
+using Oceananigans: tupleit
+
+struct SeaIceModel{GR, TD, D, CL, TS, U, T, IT, IC, STF, SMS, A} <: AbstractModel{TS}
+    grid :: GR
+    clock :: CL
+    timestepper :: TS
+    # Prognostic State
+    velocities :: U
+    tracers :: T
+    ice_thickness :: IT
+    ice_concentration :: IC
+    # Thermodynamics
+    sea_ice_thermodynamics :: TD
```

Comment:
pick `ice` or `sea_ice`. now we have `ice_thickness` but `sea_ice_thermodynamics`

Reply:
It may make sense to use `sea_ice` to refer to ice that is a mixture of brine and pure ice (eg with a salinity) and then to reserve `ice` for pure ice.

Reply:
@simone-silvestri did you address this?

Reply:
yep